src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
-import { Home, About, Contact, Navigation } from './components';
+import {Home, About, Contact, Navigation} from './components';
+import {Route} from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path="/" component={Home} />
+    <Route exact path="/about" component={About} />
+    <Route exact path="/contact" component={Contact} />
```

Comment:
Do you need `exact` for `/about` and `/contact` here? There's nothing extending off of those routes, so it seems unnecessary.